### DISCUSSION TITLE
Org repos dont show in the API

### DISCUSSION THREAD

**Discussion by Snazzah** [community/community#21888] [New to GitHub]
<p>It turns out that one of my organization’s repositories (SnazzahDI) doesn’t show up on the <code>/user/repos</code> API endpoint. I tried publicizing myself in the repository, Not sure where to go from there. I used a different access token and it’s affected across all tokens.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/snazzah">@snazzah</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Regarding this repository not showing up at this endpoint, are you a direct collaborator of this repository?</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Cheers!</p>

**Comment by Snazzah** [comment, marked as answer]
<p>I was able to find the issue. Didn’t see the thing about pagination.</p>
